maven-compat/src/main/java/org/apache/maven/repository/legacy/DefaultWagonManager.java

```diff
@@ -25,13 +25,18 @@
 import java.io.File;
 import java.io.IOException;
 import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
 import java.security.NoSuchAlgorithmException;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 
+import org.apache.commons.io.FileUtils;
```

Comment:
Can we inline the calls to `FileUtils` to not add a dependency to _commons-io_ ?